instruction-object.h

```diff
@@ -0,0 +1,138 @@
+//This is the header file for the instruction object (ex: lda, 0x102a, #0x80)
+//each line in a block of code will be an instruction object
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <regex>
+
+using namespace std;
+
+//INSTRUCTION OBJECT CLASS
+class instructionObj{
+  //class type
+  public:
+
+  //instance variables
+  std::vector<string> variables; //all variables in instruction line (ex: var1, var27, etc..)
+  std::vector<string> values; //all values in instruction line (ex: 0x102a, #0x80, al)
+  string objOpCode; //this has strings in it (ex: lda.num.mem)
+  string objGram;  //this has variables in it (ex: bcc.Var1.Var27)
+  int objNumber; //distinguishes each object from one another
+  bool noOperands; //distinuguishes if a instruction line has operands or not (ex: bcc[true] or lda al, 0x102f[false])
```

Comment:
This might be able to be put in a getter which checks the count of the operands vector and returns if there are any operands